sunpy/physics/differential_rotation.py

```diff
@@ -593,6 +593,11 @@ def differential_rotate(smap, observer=None, time=None, **diff_rot_kwargs):
     for key in all_keys:
         out_meta.pop(key)
 
+    # Check if 'rsun_ref' is in the metadata, if not, assign a default value
+    if 'rsun_ref' not in out_meta:
+        rsun_ref = out_meta.get('rsun_ref', GenericMap.rsun_meters.fget(smap))
```

Comment:
What is fget?

Reply:
rsun_meters is a property it's value is extracted via function get i.e fget

Reply:
I am not sure that this is correct.